src/balances/balances.controller.spec.ts

```diff
@@ -118,31 +118,6 @@ describe('Balances Controller (Unit)', () => {
       );
     });
 
-    describe('Config API Error', () => {
-      it('500 error if chain validation fails', async () => {
-        const chainId = '1';
-        const safeAddress = '0x0000000000000000000000000000000000000001';
-        const invalidChainResponse = {
-          ...chainFactory(chainId),
-          nativeCurrency: 1, // nativeCurrency should be an object
-        };
-        mockNetworkService.get.mockImplementation(() =>
-          Promise.resolve({ data: invalidChainResponse }),
-        );
-
-        await request(app.getHttpServer())
-          .get(`/chains/${chainId}/safes/${safeAddress}/balances/usd`)
-          .expect(500)
-          .expect({
-            message: 'Validation failed',
-            code: 42,
-            arguments: [],
-          });
-
-        expect(mockNetworkService.get.mock.calls.length).toBe(1);
-      });
-    });
```

Comment:
I still find this test valuable but given the introduction of the schema validation I would say that this should test instead what happens when the Config Service returns a `500`.

Reply:
Sure! I added another test in [d376ff0](https://github.com/5afe/safe-client-gateway-nest/pull/82/commits/d376ff082174dc6624e1eeb6ec396f4bfbc36045)